File: BikeManagerAPI/BikeManagerAPI/Models/ItemNotaEntrada.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace BikeManagerAPI.Models
{
    public partial class ItemNotaEntrada
    {
        public int CdNotaEntrada { get; set; }
        public int CdProduto { get; set; }
        public decimal VlUnitario { get; set; }
        public decimal VlTotal { get; set; }
        public decimal VlCusto { get; set; }
        public decimal QtProduto { get; set; }
        public DateTime DtRegistro { get; set; }
        public DateTime? DtAlteracao { get; set; }
        [JsonIgnore]
        public NotaEntrada NotaEntrada { get; set; }
        [JsonIgnore]
        public Produto Produto { get; set; }
    }
}

File: WebApp/src/app/cadastros/models/itemNotaEntrada.ts
import { Produto } from "./produto";
import { NotaEntrada } from "./notaEntrada";
import { Estoque } from "./estoque";

export class ItemNotaEntrada {
    cdNotaEntrada: number;
    cdProduto: number;
    //vlUnitario: number;
    vlTotal: number;
    vlCusto: number;
    qtProduto: number;
    dtRegistro: Date;
    dtAlteracao: Date;
    produto: Produto;
    notaEntrada: NotaEntrada;
    estoqueList: Array<Estoque>;
}